### ISSUE TITLE
The following assertion was thrown during a scheduler callback: 'package:flutter/src/semantics/semantics.dart': Failed assertion: line 1519 pos 12: '!owner._nodes.containsKey(id)': is not true.  

### ISSUE THREAD

**Issue by kirobass** [flutter/flutter#43559]
I have no idea on how to reproduce this. It just happened.
```
════════ Exception caught by scheduler library ═════════════════════════════════════════════════════
The following assertion was thrown during a scheduler callback:
'package:flutter/src/semantics/semantics.dart': Failed assertion: line 1519 pos 12: '!owner._nodes.containsKey(id)': is not true.


Either the assertion indicates an error in the framework itself, or we should provide substantially more information in this error message to help you determine and fix the underlying cause.
In either case, please report this assertion by filing a bug on GitHub:
  https://github.com/flutter/flutter/issues/new?template=BUG.md

When the exception was thrown, this was the stack: 
#2      SemanticsNode.attach (package:flutter/src/semantics/semantics.dart:1519:12)
#3      SemanticsNode.attach (package:flutter/src/semantics/semantics.dart:1528:15)
#4      AbstractNode.adoptChild (package:flutter/src/foundation/node.dart:132:13)
#5      SemanticsNode._replaceChildren (package:flutter/src/semantics/semantics.dart:1446:11)
#6      SemanticsNode.updateWith (package:flutter/src/semantics/semantics.dart:1878:5)
...
```

**Comment by BondarenkoStas** [comment]
@kirobass so it's not repeating error? Please provide your flutter doctor -v output.

**Comment by laibulle** [comment]
Hello, I have the same issue. My flutter doctor -v output is:

```
Waiting for another flutter command to release the startup lock...
Downloading android-arm-profile/darwin-x64 tools...                 3.8s
Downloading android-arm-release/darwin-x64 tools...                 3.2s
Downloading android-arm64-profile/darwin-x64 tools...               3.6s
Downloading android-arm64-release/darwin-x64 tools...               3.2s
Downloading android-x64-profile/darwin-x64 tools...                 3.7s
Downloading android-x64-release/darwin-x64 tools...                 3.2s
[✓] Flutter (Channel master, v1.10.15-pre.252, on Mac OS X 10.15 19A603, locale en-FR)
    • Flutter version 1.10.15-pre.252 at /Users/laibulle/flutter
    • Framework revision 903ebd714d (4 days ago), 2019-10-24 21:35:14 -0700
    • Engine revision 8d6b74aee6
    • Dart version 2.6.0 (build 2.6.0-dev.8.2 1bd6e20d76)


[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /Users/laibulle/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 29.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.1, Build version 11A1027
    • CocoaPods version 1.8.4

[✓] Android Studio (version 3.5)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 40.2.2
    • Dart plugin version 191.8593
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)

[✓] VS Code (version 1.39.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.5.1

[✓] Connected device (1 available)
    • iPhone 11 Pro Max • DAC88671-EA0E-4D5A-9CAF-72E1396956A1 • ios • com.apple.CoreSimulator.SimRuntime.iOS-13-1 (simulator)

• No issues found!
```

best regards

**Comment by BondarenkoStas** [comment]
@laibulle Can you post a small code example that reproduced the issue? Asset are put specifically to prevent some wrong actions that can lead to crash form user's side, there is high probability of an error in implementation. Or otherwise there may be some framework regression. Anyway it's not possible to check without reproduction or full stacktrace.

**Comment by laibulle** [comment]
It seems that the issue disappeared. I have wrapper all my routes into a scaffold and now it works.

```
 return MaterialApp(
      localizationsDelegates: [
        const TranslationsDelegate(),
        GlobalMaterialLocalizations.delegate,
        GlobalWidgetsLocalizations.delegate,
        GlobalCupertinoLocalizations.delegate,
      ],
      supportedLocales: [
        const Locale('fr', ''),
      ],
      theme: theme,
      title: "BookHunt",
      home: Scaffold(body: LogingScreen()),
      onGenerateRoute: Router(
        {
          '/mycopies': (context, match) => Scaffold(body: CopiesScreen()),
          '/copy/{id}': (context, match) =>
              Scaffold(body: CopyScreen(match.parameters["id"])),
          '/lookup': (context, match) => Scaffold(body: LookupScreen()),
          '/signup': (context, match) => Scaffold(body: SignUpScreen()),
          '/login': (context, match) => Scaffold(body: LogingScreen()),
          '/settings': (context, match) => Scaffold(body: SettingsScreen()),
        },
      ).get,
    );
```

**Comment by kirobass** [comment]
Hi
I am so sorry for the delay in responding. I’m on a training course right now.

From my side, I re-wrote the entire code again, but I think there is a problem with the Flutter SDK.

BR,
E

**Comment by iapicca** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. We are therefore reluctantly going to close this bug for now. Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact descriptions what happens, logs and the output of 'flutter doctor -v'  please.
All system setups can be slightly different so its always better to open new issues and reference related issues.
